### PR TITLE
add anim

### PR DESCRIPTION
**Pull request by shayb**
add animation

### CODE REVIEW

**Comment by yosriz** [comment]
@drEfrati please take it

**Comment by drEfrati** [comment]
> @drEfrati please take it

Already on it

File: kin-sendkin-lib/src/main/java/kin/sendkin/view/RecipientAddressFragment.java [hunk 1]
```diff
@@ -18,22 +16,24 @@
...
 import kin.sendkin.presenter.RecipientAddressPresenter;
 import kin.sendkin.presenter.RecipientAddressPresenterImpl;
 import kin.sendkin.presenter.SendKinPresenter;
-import kin.sendkin.core.view.SendKinEditText;
 
 public class RecipientAddressFragment extends Fragment implements RecipientAddressView {
 
     public static final String TAG = RecipientAddressFragment.class.getSimpleName();
 
+    private static final long ANIM_DELAY = 200l;
```
[the remark this answers is not in the document]
Comment (shayb) [reply kin-sendkin-lib/src/main/java/kin/sendkin/view/RecipientAddressFragment.java]: its l

File: kin-sendkin-lib/src/main/java/kin/sendkin/presenter/RecipientAddressPresenterImpl.java [hunk 2]
```diff
@@ -28,9 +28,8 @@ public void onContactChanged(boolean isEmptyList) {
 
             @Override
             public void onContactAdded(int position) {
-                getView().notifyContactChanged();
+                getView().notifyContactAdded(position);
```
[the remark this answers is not in the document]
Comment (shayb) [reply kin-sendkin-lib/src/main/java/kin/sendkin/presenter/RecipientAddressPresenterImpl.java]: its else where

**Pull request merged**